framework/rpc/grpc_testing.py

```diff
@@ -130,14 +150,57 @@ def configure(
                 )
         if app_timeout:
             request.timeout_sec = app_timeout
-        # Configure's response is empty
+        # The response is empty.
         self.call_unary_with_deadline(
             rpc="Configure",
             req=request,
             deadline_sec=timeout_sec,
             log_level=logging.INFO,
         )
 
+    def configure_unary(
+        self,
+        *,
+        metadata: Optional[ConfigureMetadata] = None,
+        app_timeout: Optional[int] = None,
+        timeout_sec: int = CONFIGURE_TIMEOUT_SEC,
```

Comment:
This should also be Optional? Same question for `configure_empty` and `configure_rpc_type`

Reply:
Yes, you're right.

Reply:
Fixed